examples/d4p_kmeans.py

```diff
@@ -1,19 +1,16 @@
 import daal4py
+import daal4py.hpat
 import hpat
 import numpy as np
 
-daal4py.daalinit(spmd=True)
-
-@hpat.jit
+@hpat.jit(nopython=True)
```

Comment:
`(nopython=True)` is not necessary since it is on by default.

Reply:
Yea, I know. Just a left-over from my desperate attempts to get the error output of actually failing features...I will remove the examples anyway...